src/index.d.ts

```diff
@@ -2,8 +2,6 @@ export type InputType = 'color' | 'date' | 'datetime' | 'datetime-local' | 'emai
 
 export type SemanticCOLORS = 'red' | 'orange' | 'yellow' | 'olive' | 'green' | 'teal' | 'blue' | 'violet' | 'purple' | 'pink' | 'brown' | 'grey' | 'black' | 'twitter' | 'google plus' | 'facebook';
```

Comment:
Looks like a bug here.  Only the true color values are available in every component (red, orange, etc).  The social colors are only available in buttons.